ios/Moonlet/Moonlet_beta-Info.plist

```diff
@@ -39,14 +39,14 @@
 			</dict>
```

Comment:
add this `Info.plist` also, it's the release plist

Reply:
`NSBluetoothAlwaysUsageDescription` has been already added to `Info.plist`
At `pod install` the file `ios/Moonlet/Moonlet_beta-Info.plist` has been rearranged